tests/integration/metrics_test.go

```diff
@@ -228,3 +234,167 @@ func TestExitedContainersWithoutTTL(t *testing.T) {
 		assert.Empty(t, sample)
 	})
 }
+
+const (
+	relativePathToTestdataFileystem = "testdata/cgroupsV1_host/"
+	InspectorContainerID            = "my-container"
+	InspectorPID                    = 666
+)
+
+var mockedTimeForAllMetricsTest = time.Date(2022, time.January, 1, 4, 3, 2, 0, time.UTC)
+
+func TestAllMetricsPresent(t *testing.T) {
+	expectedSample := biz.Sample{
+		Pids: biz.Pids{
+			Current: InspectorPID,
+			Limit:   8000,
+		},
+		Network: biz.Network{
+			RxBytes:   3402,
+			RxDropped: 2,
+			RxErrors:  10,
+			RxPackets: 28,
+			TxBytes:   5,
+			TxDropped: 2,
+			TxErrors:  3,
+			TxPackets: 4,
+		},
+		BlkIO: biz.BlkIO{
+			TotalReadCount:  39,
+			TotalWriteCount: 89,
+			TotalReadBytes:  2387968,
+			TotalWriteBytes: 50,
+		},
+		CPU: biz.CPU{
+			CPUPercent:       0.15003804623431827,
+			KernelPercent:    1.19999999,
+			UserPercent:      100,
+			UsedCores:        2.8546433726,
+			LimitCores:       float64(runtime.NumCPU()),
```

Comment:
nit: i guess that this metric is not collected from the files right? a comment might help to the future us to help understand it

Reply:
The comment is on the the mock, but I will also add it here ok.

Reply:
done